pom.xml

```diff
@@ -1,12 +1,4 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<!--********************************************************************
```

Comment:
Do we want this here? Looks like we have a mix of poms with and without licences. There are at least 48 others with it and a fair bit without.